src/executors/ubuntu.yml

```diff
@@ -26,7 +26,7 @@ parameters:
     enum: [ small, medium, medium+, large, xlarge, 2xlarge, 2xlarge+ ]
 
 docker:
-  - image: 'unityci/editor:ubuntu-<<parameters.editor_version>>-<<parameters.target_platform>>-2'
+  - image: 'unityci/editor:ubuntu-<<parameters.editor_version>>-<<parameters.target_platform>>-3.1.0'
```

Comment:
Maybe not because that would still make some unity versions unavailable to be used. The latest available in image version 3 of 2022 according to [here](https://game.ci/docs/docker/versions/) is 2022.3.16f1 which is too far from Unity LTS. Support for 2022.3.27f1 will be sufficient and it is available only under 3.1.0

Reply:
Nope, these versions in docker hub images are using glob patterns. `3` means latest `3` or `3.x.x` which means `3.1.0` as of today so I'll keep it like it was before, but bump it to 3.

I'll